core/coreobjects/include/coreobjects/property_object_impl.h

```diff
@@ -462,9 +462,9 @@ ErrCode GenericPropertyObjectImpl<PropObjInterface, Interfaces...>::getChildProp
 
 template <class PropObjInterface, class... Interfaces>
 BaseObjectPtr GenericPropertyObjectImpl<PropObjInterface, Interfaces...>::callPropertyValueWrite(const PropertyPtr& prop,
-                                                                                        const BaseObjectPtr& newValue,
-                                                                                        PropertyEventType changeType,
-                                                                                        bool isUpdating)
+                                                                                                 const BaseObjectPtr& newValue,
+                                                                                                 PropertyEventType changeType,
+                                                                                                 bool isUpdating)
 {
     if (!prop.assigned())
```

Comment:
All checks `if (prop.assigned() && ...` below this line of code are unnecessary. Maybe at this point, this is the valid assertion: `assert(prop.assigned())`